Missing the import though:
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /source/container-accesslogging/src/main/java/com/yahoo/container/logging/LogFormatter.java:[25,70] cannot find symbol
  symbol:   variable Locale
  location: class com.yahoo.container.logging.LogFormatter
[ERROR] /source/container-accesslogging/src/main/java/com/yahoo/container/logging/LogFormatter.java:[28,45] cannot find symbol
  symbol:   variable Locale
  location: class com.yahoo.container.logging.LogFormatter
[ERROR] /source/container-accesslogging/src/main/java/com/yahoo/container/logging/LogFormatter.java:[31,66] cannot find symbol
  symbol:   variable Locale
  location: class com.yahoo.container.logging.LogFormatter
[ERROR] /source/container-accesslogging/src/main/java/com/yahoo/container/logging/LogFormatter.java:[105,75] cannot find symbol
  symbol:   variable Locale
  location: class com.yahoo.container.logging.LogFormatter

--- a/container-accesslogging/src/main/java/com/yahoo/container/logging/LogFormatter.java
+++ b/container-accesslogging/src/main/java/com/yahoo/container/logging/LogFormatter.java
@@ -4,6 +4,7 @@ package com.yahoo.container.logging;
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.Locale;
 import java.util.TimeZone;
 import java.util.logging.Formatter;
 import java.util.logging.LogRecord;

